tasks/create_record.json

```diff
@@ -0,0 +1,17 @@
+{
+    "puppet_task_version": 1,
+    "supports_noop": false,
+    "description": "Create a ServiceNow record",
+    "files": ["ruby_task_helper/files/task_helper.rb"],
+    "remote": true,
+    "parameters": {
+        "table": {
+            "description": "ServiceNow table",
+            "type": "String[1]"
+        },
+        "servicenow_fields": {
```

Comment:
Why is this `servicenow_fields` instead of `fields` ? Also, I think description should be "The record's fields. This is a hash of <field_name> => <field_value>" (the `<field_name>`, `<field_value>` stuff makes the param type's meaning clearer imo).

Reply:
Sorry I misunderstood your original comment, fixed.